packages/playground/website/src/components/playground-configuration-group/index.tsx

```diff
@@ -1,302 +1,302 @@
+				<StartOverButton />
+			) : null}
+			{isResumeLastDirOpen ? (
+				<Modal
+					isOpen={isResumeLastDirOpen}
+					onRequestClose={closeResumeLastDirModal}
+					style={{
+						...defaultStyles,
+						content: {
+							...defaultStyles.content,
+							width: 550,
+						},
+					}}
+				>
+					<h2 style={{ textAlign: 'center', marginTop: 0 }}>
+						Resume working in local directory?
+					</h2>
+					<p>
+						It seems like you were working a local directory called{' '}
+						<b>"{lastDirectoryHandle!.name}"</b> during your last
+						visit. Would you like to resume working in this
+						directory or start fresh?
+					</p>
+					<form
+						onSubmit={handleResumeLastDir}
+						className={css.buttonsWrapper}
+					>
+						<Button
+							size="large"
+							onClick={() => {
+								reloadWithNewConfiguration(playground!, {
+									...initialConfiguration,
+									storage: 'temporary',
+								});
+							}}
+						>
+							Start a fresh site
+						</Button>
+						<OPFSButton
+							isMounting={mounting}
+							mountProgress={mountProgress}
+							type="submit"
+							variant="primary"
+							size="large"
+							autoFocus
+						>
+							Resume working in "{lastDirectoryHandle!.name}"
+						</OPFSButton>
+					</form>
+				</Modal>
+			) : null}
+			<Modal
+				isOpen={isOpen}
+				onRequestClose={closeModal}
+				style={{
+					...defaultStyles,
+					content: {
+						...defaultStyles.content,
+						width: 550,
+					},
+				}}
+			>
+				<PlaygroundConfigurationForm
+					currentlyRunningWordPressVersion={runningWp}
+					initialData={currentConfiguration}
+					onSubmit={handleSubmit}
+					isMountingLocalDirectory={mounting}
+					mountProgress={mountProgress}
+					onSelectLocalDirectory={
+						canUseLocalDirectory
+							? isSameOriginAsPlayground
+								? handleSelectLocalDirectory
+								: 'origin-mismatch'
+							: 'not-available'
+					}
+				/>
+			</Modal>
+		</>
+	);
+}
+
+function useCurrentlyRunningWordPressVersion(playground?: PlaygroundClient) {
+	const [
+		currentlyRunningWordPressVersion,
+		setCurrentlyRunningWordPressVersion,
+	] = useState<string | undefined>();
+
+	useEffect(() => {
+		if (playground) {
+			playground.getWordPressModuleDetails().then((details) => {
+				setCurrentlyRunningWordPressVersion(details.majorVersion);
+			});
+		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [!playground]);
+
+	return currentlyRunningWordPressVersion;
+}
+
+function useIsSameOriginAsPlayground(playground?: PlaygroundClient) {
+	const [isSameOriginAsPlayground, setIsSameOriginAsPlayground] = useState<
+		null | boolean
+	>(null);
+
+	useEffect(() => {
+		if (!playground) return;
+		(async () => {
+			setIsSameOriginAsPlayground(
+				new URL(await playground.absoluteUrl).origin ===
+					window.location.origin
+			);
+		})();
+	}, [playground]);
+
+	return isSameOriginAsPlayground;
+}

```

Comment:
is something off here with whitespace changes? I haven't investigated, but Github's diff view is getting confused, notable by how it looks normal when triggering "ignore whitespace" but without that, shows removing every line in the file and then adding them as new changes.

Reply:
I am not conscious of doing so, but somehow I changed the name of the index.tsx file to opfs-host. Then I couldn't find the index.tsx file in my repository, so I created the file from scratch, by copying all the code from the original index.tsx file, and uploaded it to my repository.

Then edit these lines: 64, 143, 147, 147, 149, 149, 177, 183 in index.tsx